File: bee1555.py
# Rafael
def func_r(x, y):
    return (3 * x) ** 2 + y ** 2
    
# Beto
def func_b(x, y):
    return 2 * x ** 2 + (5 * y) ** 2
    
# Carlos
def func_c(x, y):
    return -100 * x + y ** 3

# Número de casos de teste
n = int(input())

for i in range(n):
    # Valores de teste
    x, y = map(int, input().split(" "))

    # Resultados
    r = func_r(x, y)
    b = func_b(x, y)
    c = func_c(x, y)

    # Printar resultados
    if r > b and r > c:
        print("Rafael ganhou")
    elif b > r and b > c:
        print("Beto ganhou")
    else:
        print("Carlos ganhou")

File: bee1555.cpp
#include <iostream>
#include <cmath>
using namespace std;

// Rafael
int func_r(int x, int y) {
    return pow(3 * x, 2) + pow(y, 2);
}

// Beto
int func_b(int x, int y) {
    return 2 * pow(x, 2) + pow(5 * y, 2);
}

// Carlos
int func_c(int x, int y) {
    return -100 * x + pow(y, 3);
}

int main() {
    // Número de casos de teste
    int n;
    cin >> n;

    for (int i = 0; i < n; ++i) {
        // Valores de teste
        int x, y;
        cin >> x >> y;

        // Resultados
        int r = func_r(x, y);
        int b = func_b(x, y);
        int c = func_c(x, y);

        // Printar resultados
        if (r > b && r > c) {
            cout << "Rafael ganhou" << endl;
        } else if (b > r && b > c) {
            cout << "Beto ganhou" << endl;
        } else {
            cout << "Carlos ganhou" << endl;
        }
    }

    return 0;
}
